R/chat.R

```diff
@@ -272,13 +278,19 @@ Chat <- R6::R6Class("Chat",
 
           result <- stream_merge_chunks(private$provider, result, chunk)
         }
+        turn <- stream_turn(private$provider, result)
+
         # Ensure turns always end in a newline
         if (any_text) {
           emit("\n")
           yield("\n")
         }
 
-        turn <- stream_turn(private$provider, result)
+        if (echo == "all") {
+          is_text <- map_lgl(turn@content, S7_inherits, content_text)
+          formatted <- map_chr(turn@content[!is_text], format)
+          cat_line(formatted, prefix = "< ")
```

Comment:
There are a number of places in this file where you're using `cat_line(..., prefix = "< ")`. Should these not all be word-wrapped? (`cat_word_wrap(prefix = "< ")(formatted)`)

Reply:
I went ahead and made `cat_line` do word wrapping. If there's some reason it shouldn't, feel free to undo that part.

Reply:
Good call, thanks.